### DISCUSSION TITLE
How to remove certificates

### DISCUSSION THREAD

**Discussion by svliantiss** [nginx-proxy/acme-companion#803] [Q&A]
Hello

I'm trying to switch a website certificate from to another service for a client who ordered a certificate for themselves (no need to discuss that, they pay me).

How do I remove the certificate from let's encrypt within this docker container?

**Comment by buchdag** [comment]
Hi.

If you remove the domains from the `LETSENCRYPT_HOST` environment variable, the container [will remove symlinks to those certificates](https://github.com/nginx-proxy/docker-letsencrypt-nginx-proxy-companion/blob/004e98b3bf0d24ddf2804f112facf7fd7c1a1288/app/letsencrypt_service#L51-L119). You'll then be free to replace them with symlinks to your client certificates (which you'll have to mount inside your `nginx-proxy` / `nginx` + `docker-gen` / `letsencrypt-nginx-proxy-companion` containers, preferably as read only).
